config/src/sumeragi.rs

```diff
@@ -178,6 +221,7 @@ pub mod tests {
     use super::*;
 
     prop_compose! {
+        #[allow(unused_variables)]
```

Comment:
Not the worst way to approach this. It's fine because we have no guarantees about what the `prop_compose!` macro will do with the extra parameter.